src/components/Cta/CtaExternalLink.tsx

```diff
@@ -13,6 +13,7 @@ export const CtaExternalLink = React.forwardRef<HTMLAnchorElement, CtaExternalLi
   (props, ref) => {
     const {
       variant = 'link',
+      color = 'white',
```

Comment:
Added a color prop which could be used with some of the button variants (ghost, link etc)